Description: Don't process the missing Animal.dat image
 Upstream doesn't ship the Animal.dat file in the sources. It is a picture
 that is displayed in the "About" window.
 .
 Remove the display of that picture and resize the widgets so as to occupy
 all the space freed by it.
 .
Author: bgstack15
Author: Fab Stz <fabstz-it@yahoo.fr>
Last-Update: 2024-02-06
Forwarded: not-needed

--- a/FreeFileSync/Source/ui/gui_generated.cpp
+++ b/FreeFileSync/Source/ui/gui_generated.cpp
@@ -5567,8 +5567,8 @@
 	wxBoxSizer* bSizer183;
 	bSizer183 = new wxBoxSizer( wxHORIZONTAL );
 
-	m_bitmapAnimalSmall = new wxStaticBitmap( m_panelDonate, wxID_ANY, wxNullBitmap, wxDefaultPosition, wxDefaultSize, 0 );
-	bSizer183->Add( m_bitmapAnimalSmall, 0, wxALIGN_CENTER_VERTICAL|wxALL, 5 );
+	//m_bitmapAnimalSmall = new wxStaticBitmap( m_panelDonate, wxID_ANY, wxNullBitmap, wxDefaultPosition, wxDefaultSize, 0 );
+	//bSizer183->Add( m_bitmapAnimalSmall, 0, wxALIGN_CENTER_VERTICAL|wxALL, 5 );
 
 	wxPanel* m_panel39;
 	m_panel39 = new wxPanel( m_panelDonate, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL );
@@ -5581,13 +5581,13 @@
 	m_staticTextDonate->Wrap( -1 );
 	m_staticTextDonate->SetForegroundColour( wxColour( 0, 0, 0 ) );
 
-	bSizer184->Add( m_staticTextDonate, 0, wxALIGN_CENTER_HORIZONTAL|wxLEFT|wxALIGN_CENTER_VERTICAL, 10 );
+	bSizer184->Add( m_staticTextDonate, 0, wxALIGN_CENTER_HORIZONTAL|wxLEFT|wxRIGHT|wxALIGN_CENTER_VERTICAL, 10 );
 
 
 	m_panel39->SetSizer( bSizer184 );
 	m_panel39->Layout();
 	bSizer184->Fit( m_panel39 );
-	bSizer183->Add( m_panel39, 1, wxTOP|wxBOTTOM|wxRIGHT|wxEXPAND, 5 );
+	bSizer183->Add( m_panel39, 1, wxLEFT|wxTOP|wxBOTTOM|wxRIGHT|wxEXPAND, 5 );
 
 
 	m_panelDonate->SetSizer( bSizer183 );
--- a/FreeFileSync/Source/ui/small_dlgs.cpp
+++ b/FreeFileSync/Source/ui/small_dlgs.cpp
@@ -139,9 +139,9 @@
 
     wxImage::AddHandler(new wxJPEGHandler /*ownership passed*/); //activate support for .jpg files
 
-    wxImage animalImg(utfTo<wxString>(appendPath(getResourceDirPath(), Zstr("Animal.dat"))), wxBITMAP_TYPE_JPEG);
-    convertToVanillaImage(animalImg);
-    assert(animalImg.IsOk());
+    //wxImage animalImg(utfTo<wxString>(appendPath(getResourceDirPath(), Zstr("Animal.dat"))), wxBITMAP_TYPE_JPEG);
+    //convertToVanillaImage(animalImg);
+    //assert(animalImg.IsOk());
 
     //--------------------------------------------------------------------------
     //have animal + text match *final* dialog width
@@ -152,13 +152,13 @@
 #endif
 
     {
-        const int imageWidth = (m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 /* grey border*/) / 2;
-        const int textWidth  =  m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 - imageWidth;
+        //const int imageWidth = (m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 /* grey border*/) / 2;
+        const int textWidth  =  m_panelDonate->GetSize().GetWidth();
 
-        setImage(*m_bitmapAnimalSmall, shrinkImage(animalImg, wxsizeToScreen(imageWidth), -1 /*maxHeight*/));
+        // setImage(*m_bitmapAnimalSmall, shrinkImage(animalImg, wxsizeToScreen(imageWidth), -1 /*maxHeight*/));
 
         m_staticTextDonate->Show();
-        m_staticTextDonate->Wrap(textWidth - 10 /*left gap*/); //wrap *after* changing font size
+        m_staticTextDonate->Wrap(textWidth - 20); //wrap *after* changing font size
     }
     //--------------------------------------------------------------------------
 
